bin/check-latest-npm.js

```diff
@@ -6,12 +6,20 @@
 const { green, red, yellow } = require( 'chalk' );
 const { get } = require( 'https' );
 const { spawn } = require( 'child_process' );
+const semver = require( 'semver' );
+
+/**
+ * Internal dependencies
+ */
+const {
+	engines: { npm: npmSemver },
+} = require( '../package.json' );
 
 /**
  * Returns a promise resolving with the version number of the latest available
- * version of NPM.
+ * version of npm.
```

Comment:
Changed from `NPM` to `npm` since it's actually supposed to be [all-lowercase](https://www.npmjs.com/about) 😅 .